docs/how-to-use/search/advanced-search-user-docs.md

```diff
@@ -0,0 +1,65 @@
+# Advanced Search Component
+
+## Introduction
+
+The `AdvancedSearchComponent` allows for the creation of complex queries using a graphical widget.
+The widget's contents are then turned into a string representing a Gravsearch (SPAQRL) query to be sent to DSP-API.
```

Comment:
typo: SPARQL

Reply:
fixed in https://github.com/dasch-swiss/dsp-ui-lib/pull/295/commits/8d3c3f20a65273bfdbca05eb12bdd410426edd19